api/v1beta1/gcpmachine_types.go

```diff
@@ -197,6 +197,14 @@ type GCPMachineSpec struct {
 	// +optional
 	AdditionalNetworkTags []string `json:"additionalNetworkTags,omitempty"`
 
+	// ResourceManagerTags is an optional set of tags to apply to GCP resources managed
```

Comment:
Apologies for bring this up again.
I feel, lower camel case should be used here, the way corresponding field names appear in CRD and `kubectl explain`.

Reply:
In favor of using upper case letters, it is what is used in similar fields in the struct, like `AdditionalNetworkTags` or `AdditionalLabels`.